utils/utils.js

```diff
@@ -48,23 +48,23 @@ const utils = {
       !input
         .trim()
         .toLowerCase()
-        .match(/^[a-zA-Z]+$/g)
+        .match(/^[a-zA-Z0-9_]+$/g)
     ) {
       throw new Error(
-        `${inputName} can not have spaces and contains only letters`
-      );
-    }
-    if (
-      inputName === "title" &&
-      !input
-        .trim()
-        .toLowerCase()
-        .match(/^(?![\d])[\w\s]+$/gi)
```

Comment:
needs to removed from everywhere 
![image](https://user-images.githubusercontent.com/91623753/236294518-11d4e62b-9001-4bf1-9114-90d69e924ec2.png)